CMakeLists.txt

```diff
@@ -527,6 +527,8 @@ OCV_OPTION(OPENCV_SEMIHOSTING         "Build the library for semihosting target
 OCV_OPTION(ENABLE_PYLINT              "Add target with Pylint checks"                            (BUILD_DOCS OR BUILD_EXAMPLES) IF (NOT CMAKE_CROSSCOMPILING AND NOT APPLE_FRAMEWORK) )
 OCV_OPTION(ENABLE_FLAKE8              "Add target with Python flake8 checker"                    (BUILD_DOCS OR BUILD_EXAMPLES) IF (NOT CMAKE_CROSSCOMPILING AND NOT APPLE_FRAMEWORK) )
 
+OCV_OPTION(OPENCV_USE_LAPACK_PREFIX   "Use LAPACK_ prefix for function calls with lapack >= 3.9.1." OFF)
```

Comment:
What do you think about setting it to "ON" based on detected lapack version (to ease downstream maintenance work)?

Reply:
Sounds great to me! Just haven't figured out how to do it